src/System/Wlog/Launcher.hs

```diff
@@ -101,7 +100,7 @@ setupLogging mTimeFunction LoggerConfig{..} = do
             setSeveritiesMaybe parent (_ltSeverity)
 
         forM_ _ltFiles $ \HandlerWrap{..} -> liftIO $ do
-            let fileSeverities   = (_ltSeverity) ?: debugPlus
+            let fileSeverities   = fromMaybe debugPlus (_ltSeverity)
```

Comment:
`()` are redundant here. Could you please run `HLint` after all your changes? Before that you should copy `HLint.hs` from `cardano-sl` repository to have `universum` specific lints as well.